src/shared/modules/settings/settingsDuck.ts

```diff
@@ -97,13 +100,14 @@ const initialState = {
   maxFrames: 30,
   codeFontLigatures: true,
   editorAutocomplete: true,
+  useBoltRouting: false,
```

Comment:
This was just forgotten in the init state?

Reply:
Yup